openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -646,8 +654,12 @@ def _create_course(self, course_price, course_run_count=1):
 
         run = CourseRunFactory(key=unicode(course.id), seats=[SeatFactory(price=course_price)])
         course_runs.append(run)
+    if make_entitlement:
+        entitlements = [EntitlementFactory()]
+    else:
+        entitlements = []
```

Comment:
Nit: this could be shortened to one line.
`entitlements = [EntitlementFactory()] if make_entitlement else []`